framework/src/main/java/org/apache/drill/test/framework/TestDriver.java

```diff
@@ -179,6 +179,9 @@ public void generateReports(List<DrillTest> tests, int iteration) {
       if(drillReportsDir == null){
         drillReportsDir = CWD;
       }
+      if(drillReportsDFSDir == null){
```

Comment:
Good point. I was trying to keep drillReportsDFSDir check consistent with drillReportsDir. Have updated it accordingly.